mitmproxy/master.py

```diff
@@ -118,6 +119,37 @@ def shutdown(self):
         self.should_exit.set()
         self.addons.done()
 
+    def is_alpn_missing(self, level='error'):
```

Comment:
Can we move this into an addon that just has a `start` function to do the checks and logging?